docs/src/routes/docs/components/dropdown.svx

```diff
@@ -0,0 +1,112 @@
+---
+name: Dropdown
+relatedComponents:
+  - name: DropdownShell
+    link: ./docs/components/dropdown-shell
+shadowRaised: '0 2px 4px -1px rgba(0, 0, 0, .2), 0 3px 5px 0 rgba(0, 0, 0, .14), 0 1px 10px 0 rgba(0, 0, 0, .12);'
+---
+
+<script>
+  import { DropdownShell, Dropdown, Button } from 'attractions';
+  import { ChevronDownIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import ShadowPreview from 'src/components/docs/shadow-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+</script>
+
+# Dropdown
+
+Toggleable, contextual overlays for displaying additional details or actions. Must be wrapped with a [`DropdownShell`](./docs/components/dropdown-shell) for correct behavior.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <DropdownShell let:toggle>
+      <Button on:click={toggle}>
+        I toggle a dropdown
+        <ChevronDownIcon size="24" class="ml dropdown-chevron" />
+      </Button>
+      <Dropdown>
+        <div class="padded">
+          and I am the dropdown content
+        </div>
+      </Dropdown>
+    </DropdownShell>
+    <DropdownShell let:toggle>
+      <Button on:click={toggle}>
+        look up there
+      </Button>
+      <Dropdown top>
+        <div class="padded">
+          right here
+        </div>
+      </Dropdown>
+    </DropdownShell>
+  </div>
+  <div slot="source">
+
+```svelte
+<DropdownShell let:toggle>
+  <Button on:click={toggle}>
+    I toggle a dropdown
+    <ChevronDownIcon size="24" class="ml dropdown-chevron" />
+  </Button>
+  <Dropdown>
+    <div class="padded">
+      and I am the dropdown content
+    </div>
+  </Dropdown>
+</DropdownShell>
+<DropdownShell let:toggle>
+  <Button on:click={toggle}>
+    look up there
+  </Button>
+  <Dropdown top>
+    <div class="padded">
+      right here
+    </div>
+  </Dropdown>
+</DropdownShell>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Style Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`right`** | `false` | `boolean` | Aligns the dropdown content with the right edge of the shell. |
+| **`top`** | `false` | `boolean` | Aligns the dropdown content with the top edge of the shell. |
```

Comment:
I actually kinda think the previous version sounded better over here, saying that the dropdown is to the top of the shell. This can be interpreted as if the dropdown starts at the top edge of the shell, but still extends down

Reply:
Do you mean that the previous comment applied only to the `right` prop (and it is better now), but not to the `top`?

Reply:
Yeah, the `top` was fine as you initally wrote it